test/recovery.test.ts

```diff
@@ -0,0 +1,107 @@
+import axios from "axios";
+import { xdr } from "stellar-sdk";
+
+import { SigningKeypair, Wallet } from "../src";
+
+const testingAccountKp = SigningKeypair.fromSecret(
+  "SDZZHNNOHOLFCAQ7XZZREXTXFTEPPT3L527WB2LVYCXODEDGTT6KBUSL",
+);
+
+const accountSignerKp = SigningKeypair.fromSecret(
+  "SDZF2OUDSU32XIQYVO53X2P4F7VYP72HJP7JR3RWTT3AFZSADENNL7YZ",
+);
+
+const sponsorAccountKp = SigningKeypair.fromSecret(
+  "SCIKQPLKAARVTUX76R3PPJ5PY5KANAJ4H5TXKBAZA4L2JIQCHVGVFFGS",
+);
+
+let wallet: Wallet;
+
+describe("Recovery / Register Signers", () => {
```

Comment:
thanks for adding these tests. Imo the code still needs some more tests, for example for the public functions to make sure they're not breaking as the codebase grows(`createRecoverableWallet`, `getAccountInfo`, `replaceDeviceKey`, `signWithRecoveryServers`)

i think it would be a bit of work to mock the http requests though. Maybe we should add another ticket to do wdyt?

Reply:
oh I just noticed we have this ticket already actually:

https://stellarorg.atlassian.net/browse/WAL-572?atlOrigin=eyJpIjoiMmU2Yjk2MTFiNDAxNGUxZTliYjdjMjVjZDZjNjJiNGIiLCJwIjoiaiJ9

Reply:
@acharb yeah I totally agree this needs more tests. As this PR is opened for a long time (I was OOO last week) I'm inclined to merge it soon and tackle further testing on the ticket you linked above. What do you think?

Reply:
ya that sgtm 👍